c_glib/README.md

```diff
@@ -94,6 +96,29 @@ Others:
 % sudo make install
 ```
 
+If you use Meson, you can build and install Arrow GLib after by the followings:
+
+macOS:
+
+```console
+% cd c_glib
+% brew bundle
+% mkdir -p build
```

Comment:
We don't need to create build directory. Meson creates it automatically. :-)

Reply:
I see. I've removed `mkdir -p build`.